extensions/amp-form/amp-form.md

```diff
@@ -640,6 +674,8 @@ to the input element.
 
 For more examples, see [examples/forms.amp.html](../../examples/forms.amp.html).
 
+[filter formats="websites, ads"]
```

Comment:
This should probably be moved above `### Redirecting after a submission` to cover everything not supported in emails.